src/js/node/stream.js

```diff
@@ -3327,7 +3327,7 @@ var require_readable = __commonJS({
 
         streamReadable.pause();
 
-        const cleanup = finished(streamReadable, error => {
+        const cleanup = eos(streamReadable, error => {
```

Comment:
How intentional is this on a scale of 1 to 10

Reply:
i looked at the place we copied this from
```
import { eos as finished } from ...
```

so 10